addons/website/views/snippets/snippets.xml

```diff
@@ -111,7 +111,7 @@
                     <t id="sale_products_hook"/>
                     <t t-snippet="website.s_searchbar" t-thumbnail="/website/static/src/img/snippets_thumbs/s_searchbar.svg"/>
                     <t id="blog_posts_hook"/>
-                    <t id="event_local_events_hook"/>
+                    <t id="dynamic_event_hook"/>
```

Comment:
``event_dynamic_hook`` maybe ? to prefix with module ?

I would update dynamic_event naming to event_dynamic everywhere. Or event_list ? Because feature is more about event list display, bieng dynamic is the behavior :) 

Edit: or ``event_upcoming_snippet_hook`` for ``event_upcoming_snippet`` ?